src/components/Modals/AssetSelectorModal/DefaultAssetList.js

```diff
@@ -117,6 +115,8 @@ const DefaultAssetList = ({ tokens, collectibles, onSelectToken, onSelectCollect
       keyboardShouldPersistTaps="always"
       ListEmptyComponent={renderEmptyState()}
       contentInsetAdjustmentBehavior="scrollableAxes"
+      removeClippedSubviews
+      scrollEventThrottle={1}
```

Comment:
Interesting

Reply:
Actually, this is used for large lists smoothly scroll.